src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const DAY_RENT = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+
+  if (days < 3) {
```

Comment:
Consider following the checklist https://github.com/SerMamchur/js_task-transportation-on-vacation/blob/master/checklist.md